implementations/rust/ockam/ockam_core/src/compat.rs

```diff
@@ -0,0 +1,178 @@
+//! A facade around the various collections and primitives needed
+//! when using "std", "no_std + alloc" or "no_std" targets.
+//!
+//! When importing from the standard library:
+//!
+//!   1. always prefer core::<mod> over std::<mod> where it's
```

Comment:
Would be great if we could create automated check for this. @mrinalwadhwa @metaclips